recipes/kart/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+if [ ! -d $PREFIX/bin ] ; then
+  mkdir $PREFIX/bin
+fi
+
+make Compiler=$CXX CXX=$CXX CC=$CC LDFLAGS="-L${PREFIX}/lib" CFLAGS="-I${PREFIX}/include -L${PREFIX}/lib" CXXFLAGS="-I${PREFIX}/include -L${PREFIX}/lib"
```

Comment:
The compiler activation scripts also set `{LD,C,CXX}FLAGS` which also set the `-L` and `-I` (or rather, as of recently ,`-isystem`) for the `include`/`lib` paths.
You should be able to use
```suggestion
make Compiler=$CXX CXX=$CXX CC=$CC LDFLAGS="${LDFLAGS}" CFLAGS="${CFLAGS}" CXXFLAGS="${CXXFLAGS}"
```
If the `-L` is in fact somehow expected to be part of the `{C,CXX}FLAGS`, then
```suggestion
make Compiler=$CXX CXX=$CXX CC=$CC LDFLAGS="${LDFLAGS}" CFLAGS="${CFLAGS} ${LDFLAGS}" CXXFLAGS="${CXXFLAGS} ${LDFLAGS}"
```
or (just in case the above fails)
```suggestion
make Compiler=$CXX CXX=$CXX CC=$CC LDFLAGS="${LDFLAGS}" CFLAGS="${CFLAGS} -L${PREFIX}/lib" CXXFLAGS="${CXXFLAGS} -L${PREFIX}/lib"
```
might work.

Reply:
Argh, looks like https://github.com/hsinnan75/Kart/blob/2.5.2/src/BWT_Index/makefile#L14 is no fan of using `LDFLAGS`. We have to use one of the other suggestions then. (Or patch the makefile, or do a separate `make` of `bwt_index` with the altered flags. But I'd assume injecting it into `{C,CXX}FLAGS` is most convenient for you.)

Reply:
Please reuse the flags the compiler activation scripts set. They ensure compatibility between libraries as well as with the minimum system requirements for the users.
Try
```
make Compiler=$CXX CXX=$CXX CC=$CC LDFLAGS="${LDFLAGS}" CFLAGS="${CFLAGS} ${LDFLAGS}" CXXFLAGS="${CXXFLAGS} ${LDFLAGS}"
```
and if that does not work
```
make Compiler=$CXX CXX=$CXX CC=$CC LDFLAGS="${LDFLAGS}" CFLAGS="${CFLAGS} -L${PREFIX}/lib" CXXFLAGS="${CXXFLAGS} -L${PREFIX}/lib"
```

Reply:
I've fixed the problem and updated build.sh using LDFLAGS="${LDFLAGS}" CFLAGS="${CFLAGS}" CXXFLAGS="${CXXFLAGS}"